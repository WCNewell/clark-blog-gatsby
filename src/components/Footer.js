import React from 'react';

import LinkedIn from '../assets/linkedin.inline.svg';
import Github from '../assets/github.inline.svg';
import Twitter from '../assets/twitter.inline.svg';
import Instagram from '../assets/instagram.inline.svg';

const Footer = () => {
  return (
      <>
        <div className='footer'>
            <h5 className='footer-text'>
                Â© {new Date().getFullYear()} Clark Newell, Built with
                {` `}
                <a href="https://www.gatsbyjs.org">Gatsby</a>
            </h5>
            <a href='https://linkedin.com/in/william-clark-newell/'>
                <LinkedIn className='social-icons' alt='linkedin icon'/>
            </a>
            <a href='https://github.com/WCNewell'>
                <Github className='social-icons' alt='github icon'/>
            </a>
            <a href='https://instagram.com/willcnewell/'>
                <Instagram className='social-icons' alt='instagram icon'/>
            </a>
            <a href='https://twitter.com/WCNewell'>
                <Twitter className='social-icons' alt='twitter icon'/>
            </a>
        </div>
      </>
  );
}

export default Footer;