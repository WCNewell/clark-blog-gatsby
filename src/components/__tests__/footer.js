import React from 'react';
import renderer from 'react-test-renderer'

import Footer from '../Footer';

describe("Footer", () => {
  const tree = renderer
  .create(<Footer />)
  .toJSON()
  expect(tree).toMatchSnapshot()
});

