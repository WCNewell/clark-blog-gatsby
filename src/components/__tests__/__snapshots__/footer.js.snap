// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="footer"
>
  <h4
    className="footer-text"
  >
    Â© 
    2021
     Clark Newell, Built with
     
    <a
      aria-label="gatsby"
      className="footer-link"
      href="https://www.gatsbyjs.org"
    >
      Gatsby
    </a>
  </h4>
  <div
    className="sc-bdvvtL esfeSi"
  >
    <a
      aria-label="linked in"
      href="https://linkedin.com/in/william-clark-newell/"
    >
      <test-file-stub
        alt="linked in icon"
        className="social-icons"
      />
    </a>
    <a
      aria-label="github"
      href="https://github.com/WCNewell"
    >
      <test-file-stub
        alt="github icon"
        className="social-icons"
      />
    </a>
    <a
      aria-label="instagram"
      href="https://instagram.com/willcnewell/"
    >
      <test-file-stub
        alt="instagram icon"
        className="social-icons"
      />
    </a>
    <a
      aria-label="twitter"
      href="https://twitter.com/WCNewell"
    >
      <test-file-stub
        alt="twitter icon"
        className="social-icons"
      />
    </a>
  </div>
</div>
`;
